kura/org.eclipse.kura.core/META-INF/MANIFEST.MF

```diff
@@ -6,16 +6,22 @@ Bundle-Version: 1.0.600.qualifier
 Bundle-Vendor: Eclipse Kura
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Export-Package: org.eclipse.kura.core.data;version="1.0.0",
- org.eclipse.kura.core.linux.util;version="1.1.0",
+ org.eclipse.kura.core.linux.executor;version="1.0.0",
```

Comment:
Maybe I'm terribly wrong, but do we need to export this package? Since we moved to a new convention, maybe a better package name would be: org.eclipse.kura.core.internal.executor?

Reply:
Why I'd export an internal package? I created this package containing all the methods needed by other classes to correctly use the executor service.